docs/programming-guide.md

```diff
@@ -1353,7 +1353,7 @@ data.map(x -> accum.add(x); f(x););
 <div data-lang="python"  markdown="1">
 {% highlight python %}
 accum = sc.accumulator(0)
-data.map(lambda x => acc.add(x); f(x))
+data.map(lambda x: acc.add(x) and f(x))
```

Comment:
Clearly this has to change. I don't know much Python so would be interested to hear the answer myself: is it certain that `acc.add(x)` evaluates to true so that `f(x)` is always evaluated?